curated_transformers/tokenization/wordpiece_encoder.py

```diff
@@ -26,6 +27,27 @@ def deserialize_my_custom_class(
     return WordPieceProcessor(value.decode("utf8").split("\n"))
 
 
+def build_bert_wordpiece_encoder() -> Tok2PiecesModelT:
+    """Construct a WordPiece piece encoder model that accepts a list
```

Comment:
Let's add a short note explaining why we need a separate builder for BERT models in this docstring.

Reply:
👍 done